style.css

```diff
@@ -0,0 +1,672 @@
+:root {
+    --black:#0000;
+    --white: #fff;
+    --dark-blue: #212529;
+    --light-green:#e1f7f5f8;
+    --gray1: #d6d6d6;
+    --gray2:#9c9c9c;
+    --light: #efefef;
+    --indigo: #6610f2;
+    --gray-dark: #343a40;
+    --cyan: #17a2b8;
+    --grey-green:#308195;
+
+  }
+
+html {
+    font-family: sans-serif;
+    line-height: 1.15;
+}
+
+*,
+::after,
+::before {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: Helvetica, sans-serif;
+    margin: 0;
+    font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji;
+    font-size: 1rem;
```

Comment:
++ zastosowanie rem do font-size. Chociaż 1rem to tutaj jest domyślny, nic by nie zmieniło jego usunięcie.